news-bundle/src/EventListener/DataContainer/LayoutListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\NewsBundle\EventListener\DataContainer;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\ServiceAnnotation\Callback;
+use Contao\NewsBundle\Controller\NewsFeedController;
+use Contao\PageModel;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class LayoutListener
+{
+    public function __construct(private ContaoFramework $framework, private TranslatorInterface $translator)
+    {
+    }
+
+    /**
+     * @Callback(table="tl_layout", target="fields.newsfeeds.options")
+     */
+    public function getAllFeeds(): array
```

Comment:
```suggestion
    public function __invoke(): array
```

And the annotation should be moved to the class instead of the method

Reply:
Fixed in c65af05e4